config/initializers/rolify.rb

```diff
@@ -4,5 +4,5 @@
 
   # Dynamic shortcuts for User class (user.is_admin? like methods). Default is: false
   # Enable this feature _after_ running rake db:migrate as it relies on the roles table
-  config.use_dynamic_shortcuts
+  # config.use_dynamic_shortcuts
```

Comment:
Yes, which are some extra methods rolify provides, such as:
user.is_organizer_of? Conference.last

But we are using the simple method:
user.has_role? :organizer, Conference.last